composer.json

```diff
@@ -7,83 +7,96 @@
         "ext-openssl": "*",
-        "ext-zip": "*",
         "ext-pdo_mysql": "*",
+        "ext-simplexml": "*",
         "ext-soap": "*",
+        "ext-spl": "*",
+        "ext-xsl": "*",
+        "ext-zip": "*",
+        "lib-libxml": "*",
+        "braintree/braintree_php": "3.22.0",
+        "colinmollenhour/cache-backend-file": "1.4",
+        "colinmollenhour/cache-backend-redis": "1.10.2",
+        "colinmollenhour/credis": "1.6",
+        "colinmollenhour/php-redis-session-abstract": "~1.2.2",
+        "composer/composer": "1.4.1",
+        "magento/composer": "~1.2.0",
+        "magento/magento-composer-installer": ">=0.1.11",
+        "magento/zendframework1": "~1.13.0",
+        "monolog/monolog": "^1.17",
+        "oyejorge/less.php": "~1.7.0",
+        "pelago/emogrifier": "1.2.0",
+        "phpseclib/phpseclib": "2.0.*",
+        "ramsey/uuid": "3.6.1",
         "sjparkinson/static-review": "~4.1",
-        "ramsey/uuid": "3.6.1"
+        "symfony/console": "~2.3, !=2.7.0",
+        "symfony/event-dispatcher": "~2.1",
+        "symfony/process": "~2.1",
+        "tedivm/jshrink": "~1.1.0",
+        "tubalmartin/cssmin": "4.1.0",
+        "zendframework/zend-captcha": "^2.7.1",
+        "zendframework/zend-code": "^3.1.0",
+        "zendframework/zend-config": "^2.6.0",
+        "zendframework/zend-console": "^2.6.0",
+        "zendframework/zend-crypt": "^2.6.0",
+        "zendframework/zend-db": "^2.8.2",
+        "zendframework/zend-di": "^2.6.1",
+        "zendframework/zend-eventmanager": "^2.6.3",
+        "zendframework/zend-form": "^2.10.0",
+        "zendframework/zend-http": "^2.6.0",
+        "zendframework/zend-i18n": "^2.7.3",
+        "zendframework/zend-json": "^2.6.1",
+        "zendframework/zend-log": "^2.9.1",
+        "zendframework/zend-mail": "^2.8.0",
+        "zendframework/zend-modulemanager": "^2.7",
+        "zendframework/zend-mvc": "~2.6.3",
+        "zendframework/zend-serializer": "^2.7.2",
+        "zendframework/zend-server": "^2.6.1",
+        "zendframework/zend-servicemanager": "^2.7.8",
+        "zendframework/zend-session": "^2.7.3",
+        "zendframework/zend-soap": "^2.6.0",
+        "zendframework/zend-stdlib": "^2.7.7",
+        "zendframework/zend-text": "^2.6.0",
+        "zendframework/zend-uri": "^2.5.1",
+        "zendframework/zend-validator": "^2.6.0",
+        "zendframework/zend-view": "^2.8.1"
     },
     "require-dev": {
-        "phpunit/phpunit": "~6.2.0",
-        "squizlabs/php_codesniffer": "3.0.1",
-        "phpmd/phpmd": "@stable",
-        "pdepend/pdepend": "2.5.0",
         "friendsofphp/php-cs-fixer": "~2.1.1",
         "lusitanian/oauth": "~0.8.10",
-        "sebastian/phpcpd": "2.0.4"
+        "pdepend/pdepend": "2.5.0",
+        "phpmd/phpmd": "@stable",
+        "phpunit/phpunit": "~6.2.0",
+        "sebastian/phpcpd": "2.0.4",
+        "squizlabs/php_codesniffer": "3.0.1"
     },
     "replace": {
-        "magento/module-marketplace": "100.3.0-dev",
```

Comment:
Here's what I did after sorting packages in this section using https://plugins.jetbrains.com/plugin/5919-lines-sorter:

* ran `composer validate`
* fixed the trailing comma issue
* ran `composer update --lock`

For reference, see:

- https://getcomposer.org/doc/03-cli.md#validate
- https://getcomposer.org/doc/03-cli.md#update